File: src/main/java/smart4aviation/controller/TestController.java
package smart4aviation.controller;

import lombok.AllArgsConstructor;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;
import smart4aviation.entity.Flight;
import smart4aviation.entity.FlightCargo;
import smart4aviation.repository.FlightCargoRepository;
import smart4aviation.repository.FlightRepository;

import java.io.IOException;
import java.util.List;

@RestController
@AllArgsConstructor
public class TestController {
    private FlightRepository flightRepository;
    private FlightCargoRepository flightCargoRepository;

    @GetMapping("/flights")
    public List<Flight> getFlights() throws IOException {
        return flightRepository.findAll();
    }

    @GetMapping("/flightCargos")
    public List<FlightCargo> getFlightCargos() throws IOException {
        return flightCargoRepository.findAll();
    }
}

File: src/main/kotlin/com/example/smart4aviation/controller/TestController.kt
package com.example.smart4aviation.controller

import com.example.smart4aviation.entity.Flight
import com.example.smart4aviation.entity.FlightCargo
import com.example.smart4aviation.repository.FlightCargoRepository
import com.example.smart4aviation.repository.FlightRepository
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("test")
class TestController(
    val flightRepository: FlightRepository,
    val flightCargoRepository: FlightCargoRepository
) {

    @GetMapping("/flights")
    fun getFlights(): List<Flight> {
        return flightRepository.findAll()
    }

    @GetMapping("/flightCargos")
    fun getFlightCargos(): List<FlightCargo> {
        return flightCargoRepository.findAll()
    }
}